### ISSUE TITLE
k-toggle - Universal design problem / VoiceOver

### ISSUE THREAD

**Issue by thomasekdahlN** [konstaui/konsta#69]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

App mypostnord

### Bug description

When the html code for k-toggle is generated, it´s printed as a label, but without the label text and the relationship between the label and the input field that´s placed inside. 

It may seem that this creates an error when you use voice over on the phone, nothing happens when you click on it.

### Expected Behavior

It should be possible to toggle a checkbox in VoiceOver mode on iOS.

### Actual Behavior

It is not possible to toggle a checkbox in VoiceOver mode on iOS.

### Konsta UI version

v0.11.1

### Platform/Target and Browser Versions

iOS

### Validations

- [X] Follow our [Code of Conduct](https://github.com/konstaui/konsta/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://konstaui.com).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Konsta UI issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by thomasekdahlN** [comment]
Is it possible to pay to have this issue solved?

**Comment by nolimits4web** [comment]
Can you give an example of how the HTML should be modified that would work correctly?

**Comment by thomasekdahlN** [comment]
On a phone with voice over, the title was the only element that was read. It was not possible to click an svg link element used as a back link. We solved this by making the back link to a back button.
On voice over, a button is always read as an button. A link need a href to be read as a link.


Thomas Ekdahl
Privat
Mobile: +47 911 43 630
